pom.xml

```diff
@@ -21,7 +21,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
-		<fhir2hpo.version>1.0.1</fhir2hpo.version>
+		<fhir2hpo.version>1.0.2-SNAPSHOT</fhir2hpo.version>
```

Comment:
I had to bump this locally, probably should revert? I didn't have 1.1 in my m2 repository.

Reply:
Yes, please don't commit snapshot dependencies!

Reply:
Where can I find the 1.0.1 artifact? It wasn't in artifactory as far as I can tell. Should we build another, or can you push one from your m2 repo? You can upload directly from in artifactory. Or I can if you send it to me.

Reply:
I don't know how 1.0.1 did not end up in artifactory since we have a Jenkins job for releases to do that work. But 1.0.2 seems to be there, and we should be able to safely upgrade. That's what the launch_scenario branch is using.